Jenkinsfile

```diff
@@ -45,14 +52,24 @@ node('java') {
         // Once we've built, archive the artifacts and the test results.
         stage('Archive Artifacts / Test Results') {
             archiveArtifacts artifacts: '**/target/*.jar, **/target/*.war, **/target/*.hpi',
-                        fingerprint: true
+                    fingerprint: true
             if (runTests) {
                 junit healthScaleFactor: 20.0, testResults: '**/target/surefire-reports/*.xml'
             }
         }
     }
 }
 
+parallel linux: {
+    node('java && linux') {
+        build.call()
+    }
+},       windows: {
+    node('java && windows') {
```

Comment:
The other thing is that artifact fingerprinting will fingerprint files from both as the same file name. I have a PR I need to submit that handles this.